extensions/data-plane/data-plane-aws-s3/src/test/java/org/eclipse/edc/connector/dataplane/aws/s3/arguments/S3DataPlaneIntegrationTestArgumentProvider.java

```diff
@@ -0,0 +1,65 @@
+/*
+ *  Copyright (c) 2023 T-Systems International GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       T-Systems International GmbH
+ *
+ */
+
+package org.eclipse.edc.connector.dataplane.aws.s3.arguments;
+
+import org.junit.jupiter.api.Named;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+
+import java.util.UUID;
+import java.util.stream.Stream;
+
+public class S3DataPlaneIntegrationTestArgumentProvider implements ArgumentsProvider {
```

Comment:
we typically keep the `ArgumentsProvider` as inner classes in the tests, to make the test more readable